Makefile

```diff
@@ -1,4 +1,4 @@
-SERVER_CONFIG = config/kinto.ini
+SERVER_CONFIG = config/example.ini
```

Comment:
Ok for the .gitignore but not ok for copying `config/example.ini` to `config/kinto.ini`
And no we don't expect kinto-dist user to create `config/kinto.ini` file but rather to create a `config/production.ini` or `/etc/kinto/ini` file.

What we want to test is that they are configuring their own config file rather than loading the default one which should be missing.